control_server/httpserver/management_os.go

```diff
@@ -0,0 +1,26 @@
+package httpserver
+
+import (
+	"baas/control_server/machines"
+	"log"
+	"net/http"
+)
+
+type TestHandler struct {
+	machineStore machines.MachineStore
+}
+
+func (t TestHandler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Any reason you are using http handlers instead of handlerfuncs? It doesn't seem like you are making complex structs.

Reply:
yes, this handler will likely need access to some sort of database (@Zedfrigg's issue) and putting a reference to that in the handler seemed to me like the best way to do that. But you are right, currently it's a bit useless.

Reply:
fixed